CMake/ITKModuleMacros.cmake

```diff
@@ -6,7 +6,6 @@ include(${_ITKModuleMacros_DIR}/ITKModuleAPI.cmake)
 include(${_ITKModuleMacros_DIR}/ITKModuleDoxygen.cmake)
 include(${_ITKModuleMacros_DIR}/ITKModuleHeaderTest.cmake)
 include(${_ITKModuleMacros_DIR}/ITKModuleKWStyleTest.cmake)
-include(${_ITKModuleMacros_DIR}/ITKModuleClangFormat.cmake)
```

Comment:
@N-Dekker Ahh, with this commit, we will not be required to "know" about existence of `clang-format`. Except for use in IDEs, such as Visual Studio 😄